frontend/src/pages/Groups/GroupPage/GroupPage.tsx

```diff
@@ -77,19 +77,18 @@ function GroupHub({ group }: GroupHubProps) {
   const { id } = useParams<{ id: string }>();
   const { goBack, push } = useHistory();
   const [leaveGroupMutation] = useMutation(LEAVE_GROUP);
-  const { profile: user} = useProfileInfo();
+  const { profile: user } = useProfileInfo();
```

Comment:
Is it convention in JS to put spaces in between curly braces and their contents when dereferencing?

Reply:
Huh I never noticed that. Probably, I just let the Prettier gods handle the formatting 🦾